src/views/profile/Detail.vue

```diff
@@ -0,0 +1,211 @@
+<script setup lang="ts">
+import { DEFAULT_NETWORK_CONFIG } from "@/helpers/config";
+import { onMounted, ref, watch } from "vue";
+import useErc725 from "@/compositions/useErc725";
+import { useRoute, RouteLocationNormalizedLoaded, useRouter } from "vue-router";
+import { createIpfsLink } from "@/utils/createLinks";
+import useWeb3 from "@/compositions/useWeb3";
+import Notifications from "@/components/Notification.vue";
+import { LSP3ProfileJSON } from "@lukso/lsp-factory.js";
+import useNotifications from "@/compositions/useNotifications";
+
+const route = useRoute();
+const { isAddress } = useWeb3();
+const { notification, clearNotification, hasNotification, setNotification } =
+  useNotifications();
+const router = useRouter();
+
+const routeData = ref<RouteLocationNormalizedLoaded>(route);
+const dataSource = ref("");
+const loading = ref(true);
+const profileData = ref<LSP3ProfileJSON>(null as unknown as LSP3ProfileJSON);
+const error = ref(null as unknown as LSP3ProfileJSON);
```

Comment:
Since error is string from what I see why not just initiate like this:

`const error = ref("");`